src/conn.c

```diff
@@ -479,14 +477,21 @@ static natsStatus
 _readOp(natsConnection *nc, natsControl *control)
 {
     natsStatus  s = NATS_OK;
-    char        buffer[DEFAULT_BUF_SIZE];
+    char        *buffer;
```

Comment:
Sorry that I did not spot that earlier: please revert changes to this function. This is used to read the INFO protocol from the server and we cannot be limited by a user setting of ioBufSize (this would cause the connection to fail).

Reply:
OK, I will revert the part of these changes.
I'm also thinking to make the name more meaningful such as `MAX_INFO_MESSAGE_SIZE`.

I just have one question: the size should be 32768, or can we shrink it?

Reply:
Leave it at 32KB for now. The INFO from server is getting bigger and bigger. Is that size a problem in the context you are planning on using the library?

Reply:
> Is that size a problem in the context you are planning on using the library?

No problem at all! 

Now I fixed this at https://github.com/nats-io/nats.c/pull/250/commits/12c1efe8fcdb3f0c81036ea56048944f7985356b.